package.json

```
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "husky": "^8.0.0"
+  },
+  "scripts": {
+    "prepare": "husky install"
```

Can i suggest that we add a command here something like `"secretScan": "git secrets --scan"`, this way we can add it to the pre-commit hook... 
```suggestion
    "prepare": "husky install",
    "secretScan": "git secrets --scan"
```